spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactory.java

```diff
@@ -579,6 +583,50 @@ public void resetConnection() {
 		}
 	}
 
+	@Override
+	public void start() {
+	}
+
+	@Override
+	public boolean isRunning() {
+		return true;
```

Comment:
no, `this.running` must there anyway.
Or we can rely on the `this.contextStopped` variable...